src/main.ts

```diff
@@ -348,7 +351,7 @@ function cloneRepo(repo: string, owner: string): string {
     // create a temp directory
     const repoPath = 'actions'
     if (fs.existsSync(repoPath)) {
-      core.info('folder already exists, deleting')
+      // core.info('folder already exists, deleting')
```

Comment:
Shall we convert that to a debug message then?

Reply:
Done